### PR TITLE
Adding release notes file for PR

### PR DESCRIPTION
**Pull request by MushMal**
*Issue #, if available:*

NOTE: This is just the release notes to discuss/comment. This won't actually be pushed in. I am also terminating the build as it's not needed.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (disa6302) [inline RELEASE_NOTES.md:1]: Need to add release version.

Comment (MushMal) [reply RELEASE_NOTES.md:1]: Cool - will do

Comment (MushMal) [reply RELEASE_NOTES.md:1]: BTW, as this is for PR purposes only, I will be collecting the data and then using it. I believe the release version goes into GitHub tool

Comment (disa6302) [reply RELEASE_NOTES.md:1]: Yes. Agreed. The notes would be going in when we cut the release tag.

**Pull request closed without merging**